plugins/env/db/migrate/20150603211116_change_env_deploy_group_to_scope.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeEnvDeployGroupToScope < ActiveRecord::Migration
+  def change
+    remove_index :environment_variables, name: "environment_variables_unique_deploy_group_id"
+    add_column :environment_variables, :scope_type, :string
+    rename_column :environment_variables, :deploy_group_id, :scope_id
+
+    EnvironmentVariable.update_all scope_type: "DeployGroup"
```

Comment:
some of our older migrations have `Model.reset_column_information` here, whatever works